AncientReligions/events/ACR_hellenic_revival_events.txt

```diff
@@ -118,6 +119,13 @@ character_event = {
 		ai_chance = {
 			factor = 15
 			modifier = {
+				factor = 5
+				host = {
+					ai = no # Make it 50% chances for the player
+					NOT = { has_character_flag = flag_courtier_studied_the_classics }
+				}
+			}
```

Comment:
Nice job with this one! Should provide a much better experience for the players.